PythonAPI/pycocotools/coco.py

```diff
@@ -286,6 +286,14 @@ def showAnns(self, anns):
                             plt.plot(x[sk],y[sk], linewidth=3, color=c)
                     plt.plot(x[v>0], y[v>0],'o',markersize=8, markerfacecolor=c, markeredgecolor='k',markeredgewidth=2)
                     plt.plot(x[v>1], y[v>1],'o',markersize=8, markerfacecolor=c, markeredgecolor=c, markeredgewidth=2)
+
+                if draw_bbox:
+                    [bbox_x, bbox_y, bbox_w, bbox_h] = ann['bbox']
+                    poly = [[bbox_x, bbox_y], [bbox_x, bbox_y+bbox_h], [bbox_x+bbox_w, bbox_y+bbox_h], [bbox_x+bbox_w, bbox_y]]
+                    np_poly = np.array(poly).reshape((4,2))
+                    polygons.append(Polygon(np_poly))
+                    color.append(c)
```

Comment:
I think the color needs to change for visibility.

Reply:
How about using dashing line?
```suggestion
ax.add_patch(Polygon(np_poly, linestyle='--', facecolor='none', edgecolor=c, linewidth=2))
```
With which we can have folling output
![sample](https://user-images.githubusercontent.com/19389655/56892144-49b70e80-6ab1-11e9-9fc5-f37baad7a77a.png)

Reply:
> How about using dashing line?
> 
> ```
> ax.add_patch(Polygon(np_poly, linestyle='--', facecolor='none', edgecolor=c, linewidth=2))
> ```
> 
> With which we can have following output
> ![sample](https://user-images.githubusercontent.com/19389655/56892144-49b70e80-6ab1-11e9-9fc5-f37baad7a77a.png)

@CSU-NXY where should this line be put?